efi/meson.build

```diff
@@ -20,11 +18,7 @@ endif
 
 efi_libdir = get_option('efi-libdir')
 if efi_libdir == ''
-  cmd = 'cd /usr/lib/$(@0@ -print-multi-os-directory) && pwd'.format(efi_cc)
-  ret = run_command('sh', '-c', cmd)
-  if ret.returncode() == 0
-    efi_libdir = ret.stdout().strip()
-  endif
+  efi_libdir = '/usr/lib/' / run_command(cc.cmd_array(), '-print-multi-os-directory').stdout().strip()
```

Comment:
I know it's valis but the naked `/` trips me up every time. Would join_paths be clearer here?

Reply:
`/` is the one you're supposed to use for meson OS independent code, but I guess this really is Linux only the way it's done right now.